.github/workflows/auto_update_doc.yml

```diff
@@ -0,0 +1,45 @@
+name: Auto update documentation
+on:
+  pull_request_target:
+  workflow_dispatch:
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name == 'workflow_dispatch' }}
+    cancel-in-progress: true
+
+jobs:
+  auto-update-doc:
+    if: contains( github.event.pull_request.labels.*.name, 'auto update doc')
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write
+
+    steps:
+    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      with:
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+        # Checkout the branch made in the fork. Will automatically push changes
+        # back to this branch.
+        ref: ${{ github.head_ref }}
+    - name: Setup Python
+      uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4.7.0
+      with:
+        python-version: "3.11"
+    - name: Install ONNX from source and update documentation
+      run: |
+        python -m pip install -q --upgrade pip
+        python -m pip install -r requirements-release.txt
+        git submodule update --init --recursive
+        export ONNX_ML=1
+        pip install -e .
+        python onnx/defs/gen_doc.py
+        python onnx/gen_proto.py -l
+        python onnx/gen_proto.py -l --ml
+        python onnx/backend/test/stat_coverage.py
+        git diff -- . ':(exclude)onnx/onnx-data.proto' ':(exclude)onnx/onnx-data.proto3'
+
+    - name: Commit changes with updated documentation
+      uses: stefanzweifel/git-auto-commit-action@3ea6ae190baf489ba007f7c92608f33ce20ef04a # v4.16.0
```

Comment:
Should we fork this to the onnx org?

Reply:
May I understand why? Due to security reason?

Reply:
Or in case it was removed for some reason and stops to work. But no big deal atm I suppose